realm/src/main/java/io/realm/internal/TableQuery.java

```diff
@@ -589,16 +592,16 @@ public Date minimumDate(long columnIndex) {
         }
         return null;
     }
-    
+
     // isNull and isNotNull
     public TableQuery isNull(long columnIndices[]) {
-        validateQuery();
+        queryValidated = false;
```

Comment:
In the other methods this is below the native method call. It probably doesn't matter much, but would be nice to have it moved for consistency.